.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to cs102
+
+Keep in mind that this is a first year CS course. 
```

Comment:
Think this needs some elaboration. What can I expect students to already know, and what are some examples of topics that are considered out of scope and/or too advanced?

Related, I think a note about staying on the source topic is needed. A short description of the course objectives would probably help here.